spec/support/features/javascript_driver_helper.rb

```diff
@@ -1,5 +1,5 @@
 module JavascriptDriverHelper
   def javascript_enabled?
-    Capybara.current_driver == Capybara.javascript_driver
+    %i[headless_chrome headless_chrome_mobile].include?(Capybara.current_driver)
```

Comment:
What motivated this change?

Reply:
There are actually 2 javascript drivers. The headless chrome one we all know and love, and a headless chrome driver with a mobile window (which we use in a few specs where UI elements get shuffled around).

You can only have one registered as the default JS driver, but JS is enabled when we are using either one.